src/IssueViz.Security/Hotspots/LocalHotspotsStore.cs

```diff
@@ -172,12 +173,12 @@ private void RematchAll()
                     kvp => CreateLocalHotspots(kvp.Value.Select(localHotspot => localHotspot.Visualization)));
 
                 var visualizations = GetAll();
-                
+
                 if (!visualizations.Any())
```

Comment:
The issue could have been unreviewed before and became reviewed, but it won't get added to the event. TBF I think we don't use the payload of the event and always call GetAll, however I would still fix it so that it's correct or at least log a bug

Reply:
Actually, the event won't raise at all so this is an important issue